### PR TITLE
Update LOCALIZATION_CONTRIBUTORS

### PR DESCRIPTION
**Pull request by jonathan322021**
Adding my name to the list

### Summary

Related -- Spanish

### CODE REVIEW

File: LOCALIZATION_CONTRIBUTORS [hunk 1]
```diff
@@ -108,7 +108,7 @@ Rodrigo Rosado Rivial
 Sandra del Mar Soto Corderi
 Angel Gonzalez
 Siddhartha E Morales Guzman
-
+Jonathan Amador Lorence
```
Comment (CarlosQbit) [inline LOCALIZATION_CONTRIBUTORS:112]: Carlos Alejandro Quiroz Solís

Comment (SooluThomas) [reply LOCALIZATION_CONTRIBUTORS:112]: @CarlosQbit Please open a new pull request if you'd like to join. Let me know if you need help with that. Thank you

Comment (jonathan322021) [reply LOCALIZATION_CONTRIBUTORS:112]: It's there something I have to do from my side. I am new contributing

Comment (jonathan322021) [reply LOCALIZATION_CONTRIBUTORS:112]: My name still is not on the list

**Review by SooluThomas — APPROVED** [review]
Joined Spanish translations in Crowdin

**Pull request merged**